src/perf/lib/ThroughputServer.cpp

```diff
@@ -0,0 +1,188 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    QUIC Perf Throughput Server Implementation.
+
+--*/
+
+#ifdef QUIC_CLOG
+#include "ThroughputServer.cpp.clog.h"
+#endif
+
+#ifndef _KERNEL_MODE
+#define QUIC_TEST_APIS 1
+#endif
```

Comment:
Why? I don't see any other kernel mode specific code below.

Reply:
If you have this defined in kernel mode, it fails to build, because of what gets included behind this flag.